README.md

```diff
@@ -6,9 +6,21 @@ Machine learning model for recognizing handwritten numbers
 
 Find the Model in the release section.
 
-PreTrained Model available in onnx format. TFLITE has issues with int quantization. The CTC loss isn't supported on TFLITE interpretor. onnx-runtime can be used for model inference.
+PreTrained Model available in onnx format. TFLITE has issues with int quantization. The model on tflite version does not suppot quantization for int8. onnx-runtime can be used for model inference.
+
+## Training Steps
+
+1. Clone the repository
+2. Create a data, logs and models directory in the root
```

Comment:
@sshivaditya2019 Instead of explaining this in the documentation, can you create the directories `data`, `logs` and `models`, and remove this step from the `README`? Just to make it as easy as possible to reproduce the steps.

Reply:
Ok will add the same

Reply:
@sshivaditya2019 Or generate these folders using Python code. That would probably be the most robust solution?